src/pages/theming/dark-mode.md

```diff
@@ -0,0 +1,121 @@
+---
+previousText: 'Advanced'
+previousUrl: '/docs/theming/advanced'
+contributors:
+  - mhartington
+---
+
+# Dark Mode
```

Comment:
Add a reference to the `css/themes` directory where they can find a few examples